### ISSUE TITLE
Make a custom focus highlight that updates as particles are rubbed away

### ISSUE THREAD

**Issue by zepumph** [phetsims/friction#43]
From @terracoda's comment in #38.

>@zepumph, just adding one more thought until we have a design document.

>As rows of particles disappear, I think it might make sense to have the bottom edge follow the edge of yellow particles, but not shrink. This would mean the highlight would move up, so more yellow book space was part of the highlight, rather than the green particles being included within the focus highlight.

Currently I have tried two approaches.
1. Default focus highlight. This updates around the yellow particles and has the behavior that we want from the above the comment. The problem is that it only wraps the "dragArea," which is the rows of yellow atoms.
2. Custom focus highlight. This is great because I made it bigger than just the 'dragArea' but it is then static. 

@jessegreenberg do you have any ideas about how I could make a custom focus highlight node that changes size based on the particle rows? Have you done something like this before?

**Comment by jessegreenberg** [comment]
@zepumph one approach could be to observe changes to the model and redraw a shape and re-set the focus highlight when the bottom row is gone. Possibly something similar to the way the dragArea rectangle is adjusted in Magnifier.js?

**Comment by zepumph** [comment]
That was it! In fact it was simple once I realized that that was how the `dragArea`'s height was being adjusted, I just had to add a line to update the focus highlight too.

Here is what it looks like now: 

on start up
![image](https://user-images.githubusercontent.com/6856943/30711026-dfde70ea-9eb3-11e7-8bbf-3cbf088001aa.png)


and once all atoms have been evaporated
![image](https://user-images.githubusercontent.com/6856943/30711047-f8a1331a-9eb3-11e7-81ba-4ccc93d59f24.png)


This issue has been resolved. Thanks for the help @jessegreenberg. Closing
